org.pjbroad.EternallandsClient.json

```diff
@@ -0,0 +1,169 @@
+{
+	"app-id": "org.pjbroad.EternallandsClient",
+	"runtime": "org.freedesktop.Platform",
+	"sdk": "org.freedesktop.Sdk",
+	"runtime-version": "1.6",
+	"command": "launcher.sh",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--device=dri",
+		"--filesystem=~/.elc:create"
+	],
+
+	"cleanup" : [
+		"*.la",
+		"*.a",
+		"/include",
+		"/lib/pkgconfig",
+		"/lib/debug"
+	],
+
+	"modules": [
+		"shared-modules/glu/glu-9.0.0.json",
+		"shared-modules/SDL/SDL-1.2.15.json",
+		"shared-modules/SDL/SDL_net-1.2.8.json",
+		"shared-modules/SDL/SDL_image-1.2.12.json",
+
+		{
+			"name": "libpng",
+			"config-opts": [ "--disable-static" ],
+			"cleanup": [ "/bin", "/share" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://downloads.sourceforge.net/project/libpng/libpng16/1.6.35/libpng-1.6.35.tar.gz",
+					"sha256": "2b82ab3e996803b80bc73206857e826a155d3ebb374e8eb03a87a63c6f672cf7"
+				}
+			]
+		},
+
+		{
+			"name": "libcal3d",
+			"config-opts": [ "--disable-static" ],
+			"cleanup": [ "/bin", "/share" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/raduprv/Eternal-Lands/releases/download/1.9.5.2/cal3d-0.11.0.tar.gz",
+					"sha256": "b3a3fc80053cc00fbe7585f972c0e11669dbd2a13a371a10cc2dac7c9ed5f473"
+				},
+				{
+					"type": "patch",
+					"path": "cal3d-0.11.0-patch"
+				}
+			]
+		},
+
+		{
+			"name": "client",
+			"buildsystem": "simple",
+			"build-commands": [
+					"make -f Makefile.linux release PLATFORM=\"\" -j 4",
+					"install el.x86.linux.bin /app/bin"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/raduprv/Eternal-Lands",
+					"tag": "1.9.5.2"
+				},
+				{
+					"type": "patch",
+					"path": "client-patch"
+				}
+			]
+		},
+
+		{
+			"name": "launcher",
+			"buildsystem": "simple",
+			"build-commands": [
+					"install launcher.sh /app/bin"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"path": "launcher.sh",
+					"dest-filename": "launcher.sh"
+				}
+			]
+		},
+
+		{
+			"name": "exports",
```

Comment:
This doesn't have to be a separate module and can be part of `client`.

Reply:
I found it easier to make modifications keeping it separate to save rebuilds but I can move it.

Reply:
Makes sense.